src/sbi_cove.adoc

```diff
@@ -770,10 +770,16 @@ enum tsm_state {
     TSM_READY = 2
 };
 
-#define COVE_TSM_CAP_PROMOTE_TO_TVM       0x1
-#define COVE_TSM_CAP_LOCAL_ATTESTATION    0x2
-#define COVE_TSM_CAP_AIA                  0x3
-#define COVE_TSM_CAP_DYN_PAGE_CONVERSION  0x4
+/* Below specifies bit position in the `tsm_capabilities` field */
+#define COVE_TSM_CAP_PROMOTE_TVM         0x0
+#define COVE_TSM_CAP_ATTESTATION_LOCAL   0x1
+#define COVE_TSM_CAP_ATTESTATION_REMOTE  0x2
+#define COVE_TSM_CAP_AIA                 0x3
+#define COVE_TSM_CAP_MEMORY_ALLOCATION   0x4  
+
+/* Reserved IDs for TSM implementations */
+#define COVE_TSM_IMPL_SALUS            0x1
+#define COVE_TSM_IMPL_ACE              0x2
```

Comment:
Add a table to cover TSM implementation. It will probably have the following columns:
| Name       | Value.     |  Definition           | 
| COVE_TSM_IMPL_SALUS | 1 | Salus COVE implementation
|COVE_TSM_IMPL_ACE.     | 2 |. ACE COVE implementation
|  Reserved |  | Future COVE implementations should be assigned in this space.

or something similar, it should be consistent with other tables in the specification

Reply:
I modified it accordingly.